spec/datadog/tracing_spec.rb

```diff
@@ -8,7 +12,7 @@
     subject(:active_span) { described_class.active_span }
 
     it 'delegates to the tracer' do
-      expect(Datadog.tracer).to receive(:active_span).and_return(returned)
+      expect(described_class.send(:tracer)).to receive(:active_span).and_return(returned)
```

Comment:
I suggest expecting here that we call stuff on `Datadog.send(:components).tracer`, since the intent of the specs on this file is to validate that this module is effectively a shim for internal APIs (regardless of how this module actually gets at those APIs).

(This note applies to most of the spec)

Reply:
That makes sense.